web/sites/default/config/views.view.watchdog.yml

```diff
@@ -1090,7 +1050,15 @@ display:
     display_plugin: page
     position: 1
     display_options:
-      display_extenders: {  }
+      cache:
```

Comment:
Disabling caching on the 'Recent log messages' view because it doesn't update in realtime with new errors.